lib/portage/util/_async/AsyncFunction.py

```diff
@@ -25,6 +25,8 @@ class AsyncFunction(ForkProcess):
 
     def _start(self):
         pr, pw = multiprocessing.Pipe(duplex=False)
+        self.fd_pipes = {} if self.fd_pipes is None else self.fd_pipes
+        self.fd_pipes[pw] = pw
```

Comment:
We actually need to pass stdin via fd_pipes. See https://github.com/gentoo/portage/pull/1145.

Reply:
aha good to know, I will close this PR since its obselete, good job!
KR